src/webhook/mutation/namespace_mutator/mutator.go

```diff
@@ -73,22 +67,20 @@ func (nm *namespaceMutator) Handle(ctx context.Context, request admission.Reques
 }
 
 func decodeRequestToNamespace(request admission.Request, namespace *corev1.Namespace) error {
-	decoder, err := admission.NewDecoder(scheme.Scheme)
-	if err != nil {
-		return errors.WithStack(err)
-	}
+	decoder := admission.NewDecoder(scheme.Scheme)
 
-	err = decoder.Decode(request, namespace)
+	err := decoder.Decode(request, namespace)
 	if err != nil {
 		return errors.WithStack(err)
 	}
 	return nil
 }
 
-func newNamespaceMutator(ns string, apiReader client.Reader) admission.Handler {
+func newNamespaceMutator(ns string, apiReader client.Reader, client client.Client) admission.Handler {
```

Comment:
```suggestion
func newNamespaceMutator(client client.Client, apiReader client.Reader, namespace string) admission.Handler {
```

https://github.com/Dynatrace/dynatrace-operator/blob/main/doc/coding-style-guide.md#function-parameter-and-return-value-order